CONFIGURATION.md

```diff
@@ -0,0 +1,95 @@
+# Configuration
+
+#### Creating another superuser
+
+*This should be executed after executing all docker commands. (check: - [Running the application in local environment](#running-the-application-in-local-environment))
+
+1. Create a superuser for admin access running `python manage.py createsuperuser`.
```

Comment:
shouldn't it be like this?
1. Run `docker exec -it video-hub bash` to access the video-hub container.
2. Create a superuser for admin access running `python manage.py createsuperuser`